.github/workflows/phpunit.yml

```diff
@@ -70,4 +70,4 @@ jobs:
       - name: Run test suite
         run: |
           cd development/custom/plugins/plugin-shopware-6
-          composer test
+          SYMFONY_DEPRECATIONS_HELPER=disabled composer test
```

Comment:
Shopware internally has some deprecations, which are way too many that we can get rid of, so we need to disable the deprecation helper again, unfortunately.